ansible/roles/atmo-realvncserver/tasks/main.yml

```diff
@@ -123,6 +123,14 @@
   tags:
     - kill-vnc
 
+- name: Enable unencrypted connections to RealVNC
+  lineinfile:
+    dest: /etc/vnc/config.custom
+    state: present
+    backrefs: yes
+    regexp: "^SecurityTypes=RA2"
+    line: "SecurityTypes=RA2,VncAuth"
```

Comment:
So far, clank group_vars aren't programmatically related to atmosphere-ansible group_vars.  We do have a new pattern for optional roles/functionality in atmosphere-ansible, see [here](https://github.com/cyverse/atmosphere-ansible#configuring-atmosphere-ansible) and an example playbook [here](https://github.com/cyverse/atmosphere-ansible/blob/master/ansible/playbooks/instance_deploy/30_post_user_install.yml) (but you could also implement the conditional logic inside a role). I would call the variable `SETUP_GUACAMOLE` for consistency with the other "optional functionality" variables.